test/SerialIOTest.cpp

```diff
@@ -1121,7 +1121,7 @@ TEST_CASE( "no_serial_hdf5", "[serial][hdf5]" )
     REQUIRE(true);
 }
 #endif
-#if openPMD_HAVE_ADIOS1
+#if openPMD_HAVE_ADIOS1 && !openPMD_HAVE_MPI
```

Comment:
uff, I rebased on your branch and applied the mentioned changes.
I won't get a running solution with that approach unless I do explicit symbol hiding/exposing in the two transalation units for adios, since otherwise symbols like `adios_open` and `common_adios_open` will be mismatched at link and runtime.

Can do, but this will exclude pgic++ at least since not all compilers know symbol hiding. harsh world...

Reply:
We could maybe go for `dlopen()` for one ADIOS part xD